### PR TITLE
Add Bootstrap for Responsive Web

### PR DESCRIPTION
**Pull request by hyunjun529**
- Bootstrap compiled LibSass, and it's customized.
- legacy style.css was named style_old.css.
- Bootstrap version is 3.3.7.

---

please merge to develop branch https://github.com/monitflux/goni-dashboard/pull/10

### CODE REVIEW

File: views/index_min.ejs [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+<meta name="viewport" content="width=device-width, initial-scale=1">
 <title>Goni Dashboard</title>
 <link rel="stylesheet" href="//spoqa.github.io/spoqa-han-sans/css/SpoqaHanSans-kr.css" type="text/css" />
 <link rel="stylesheet" href="/style.css" type="text/css" />
+<link rel="stylesheet" href="/style_old.css" type="text/css" />
 <link rel="stylesheet" href="https://s3.ap-northeast-2.amazonaws.com/static.goniapm.io/reactselect.css" type="text/css" />
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.1/css/font-awesome.min.css" />
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/cal-heatmap/3.6.0/cal-heatmap.css" />
+<!--
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/flexboxgrid/6.3.0/flexboxgrid.min.css" type="text/css" />
```
Comment (devholic) [inline views/index_min.ejs]: Why do you comment this line?

File: src/style_old.css [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+  height: 50px;
+  width: 270px;
+}
+.login-button {
+  background-color: #ffffff;
+  border: 1px solid #4c80f1;
+  border-radius: 5px;
+  color: #2c5ae9;
+  height: 50px;
+  /* width: 270px; */
```
Comment (InhoChoi) [inline src/style_old.css]: If you don't use it, I think it have to remove.

File: src/style_old.css [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+  margin-top: 15px;
+}
+.login-input-username {
+  background-color: #f8fafb;
+  border-left: 1px solid #eaeeef;
+  border-right: 1px solid #eaeeef;
+  border-top: 0.5px solid #eaeeef;
+  border-bottom: 0.5px solid #eaeeef;
+  height: 50px;
+  /* width: 270px; */
```
Comment (InhoChoi) [inline src/style_old.css]: Same reason. It have to remove

File: src/style_old.css [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+}
+/*
+.dashboard {
+  background: #ffffff;
+  border: 1px solid #eaeeef;
+  margin: 20px;
+  padding: 10px;
+  min-height: 100%;
+}
+*/
```
Comment (InhoChoi) [inline src/style_old.css]: Also. Same Reason

File: views/index_min.ejs (hunk 1 above)
Comment (hyunjun529) [inline views/index_min.ejs]: [FlexboxGird](http://flexboxgrid.com/) is Bootstrap style gird system.
it's duplication with Bootstrap, so i commented.

**Comment by InhoChoi** [comment]
image
How about Settings than SETTINGS?

**Comment by hyunjun529** [comment]
### Change log

- [ ] remove unused css properties about old login
- [ ] sidebar's wrong word modify


### Test

for test, modify this file 
https://github.com/monitflux/goni/blob/master/docker/dashboard/Dockerfile
line 15, 16
`
RUN git clone https://github.com/hyunjun529/goni-dashboard && \
  cd goni-dashboard && npm i && npm run build
`

File: src/style_old.css (hunk 3 above)
Comment (hyunjun529) [inline src/style_old.css]: checked

File: src/style_old.css (hunk 2 above)
Comment (hyunjun529) [inline src/style_old.css]: checked

File: src/style_old.css (hunk 4 above)
Comment (hyunjun529) [inline src/style_old.css]: checked

File: views/index_min.ejs (hunk 1 above)
Comment (hyunjun529) [inline views/index_min.ejs]: checked, removed FlexboxGrid

**Comment by devholic** [comment]
LGTM 👍

**Pull request merged**